active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,120 @@
+the leaf node can truncated if crumbling from the root node to the leaf node's parent is not enough for a resizing scenario. 
+
+A Breadcrumb control with 3 nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+
+A Breadcrumb "crumbling" with 3 visible nodes:
+
+![Breadcrumb crumbling with 3 visible nodes](images/Breadcrumb_crumbling.PNG)
+
+A Breadcrumb control with the last node truncated:
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+The following Xaml markup  creates a simple Breadcrumbe control with three nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+ ~~~~ 
+     <muxc: Breadcrumb ItemsSource = {"Node", "Node", "Node"} /> 
+~~~~
+More examples to be added
+
+## Breadcrumb member notes
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | provides the content of the Breadcrumb | Null |
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  | N/A |
+
+Spec note: the V2 of Breadcrumb will add flyouts from chevrons to view children of a node.
+Additional properties will be added to the API to enable this functionality. 
+
+# API Details
+
+```cs
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Object Item { get; };
+}
+
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+unsealed runtimeclass BreadcrumbLayout : NonVirtualizingLayout
+{
+    BreadcrumbLayout();
+}
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
+{
+    Breadcrumb();
+
+    Object DropdownItemTemplate{ get; set; };
+    Object ItemsSource{ get; set; };
+    Object ItemTemplate{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<Breadcrumb, BreadcrumbItemClickedEventArgs> ItemClicked;
+
+    static Windows.UI.Xaml.DependencyProperty DropdownItemTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemTemplateProperty{ get; };
+}
+```
+
+# Appendix
+
+## UI Automation Pattern
+
+Breadcrumb will use the
+[InvokePattern](!https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-invoke-control-pattern).
+
+## Keyboard
+
+* The user can tab into the Breadcrumb and tab out.
+The first tab in gives focus to the first (highest-level) node.
+Once the Breadcrumb has focus, the user can use the left/right arrows to navigate from node to node.
+* If the Breadcrumb is crumbled, down/up will open the flyout from
+the ellipsis and down/up will navigate between the flyout items
+
+## Gamepad
+
+* Spatial navigation to focus and select the nodes.
+
+## Narrator
+
+When the focus is on a node, Narrator will announce the node
+as well as "n of m" to convey which node in the path it is.
```

Comment:
Would n and m be relative to the number of nodes currently visible or the total number of nodes? for example, if we have a breadcrumb with 5 items:
... > Node 4 > Node 5
where Node 1, Node 2 and Node 3 are 'crumbled', would Node 4 be announced as "4 of 5" or would it be "2 of 3". In the case of the crumbled nodes, would they also be enumerated from 1 to 3?

Reply:
Great callout. I would expect 4 of 5, but only if keyboarding will scroll through the list. 

Precedence is probably more interesting. But for example in Explorer the breadcrumb isn't actually announced as a list at all.

Reply:
what do you mean by precedence in this context? are you referring to the highest to lowest level precedence or is it something different?

Reply:
Sorry, precedence as in what is the screen reader experience in other Breadcrumb UXs today.

Reply:
Looks like the file explorer works different than breadcrumbs found in websites, in the case of file explorer, the whole list is another representation of the path, that's why when it gets focus the whole path is read, while in most websites, all breadcrumbs are considered elements in a numbered list (to keep hierarchy), also as there's no metadata for breadcrumbs, the only announcement made is that the navigation type is a breadcrumb, the big difference between our control and breadcrumbs in websites (such as the one in the [docs.microsoft.com](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.itemsrepeater.itemssourceview?view=winui-2.5) website) is that our control performs crumbling while others become multi-leveled, which could explain why no context is lost with the ordered items

![image](https://user-images.githubusercontent.com/35784165/107432956-8664c600-6add-11eb-9347-dfa039f2a024.png)

Reply:
@YuliKl any thoughts on this ?

Reply:
decision: read 'n of m' for the visible items. read 'more' for the ellipsis. within a flyout (evoked from the ellipsis or a chevron in future Breadcrumb) read 'n of m'

Reply:
updated with this behavior, also added the following details that need discussion:
  * The down/up keys will navigate between the flyout items. The ellipsis is not focusable while the flyout is open.
  * From any item in the flyout the user can use right arrow to go to the node right of the ellipsis.

Reply:
@YuliKl could you give any feedback for the additional keyboarding details (the two bullets above)?